pkg/kwokctl/runtime/binary/cluster.go

```diff
@@ -940,7 +940,7 @@ func (c *Cluster) Logs(ctx context.Context, name string, out io.Writer) error {
 		return nil
 	}
 
-	f, err := os.OpenFile(logs, os.O_RDONLY, 0640)
+	f, err := os.OpenFile(logs, os.O_RDONLY, 0o640)
```

Comment:
```suggestion
	f, err := os.OpenFile(logs, os.O_RDONLY, 0640)
```

I think it makes sense, but let's revert to it first, to keep the style consistent with the others.

We can change it all at once in the new PR with golangci to avoid these values being changed by different lint configurations.

Reply:
Okay!